app/assets/scss/_index-page.scss

```diff
@@ -155,3 +155,7 @@
     }
   }
 }
+
+.heading-xmedium {
```

Comment:
Our classes should be named for something vaguely semantic - not to do with how they look.

And in fact, our H3's (and H2's, etc) should be a consistent size across the site. So this shouldn't really be a thing... should it?

Reply:
Yep we should avoid random per-app styling and stick to the toolkit-defined styles where possible.

Also, yes to semantic naming.  Not even just "vaguely" if possible.

Reply:
Probably "Smaller" heading from here? http://alphagov.github.io/digitalmarketplace-frontend-toolkit/page-heading.html

Reply:
"smaller" is definitely not semantic _either_  (so I think my "vaguely" wasn't so far off ;-) )

Reply:
"Heading" is the semantics, "Smaller" is an attribute of that thing :P

Reply:
hmm, the problem here though is not "heading", it's "medium", which is as much a (formatting-related) attribute as is "small"... but more to the point, what is a "small h2" compared to a "regular h3"? How could that help users understand the structure of our pages...

Reply:
Yes, point taken :)
I think the main thing is that we should be using frontend toolkit elements for everything we possibly can.

Reply:
I can call it `subheading` ?